### ISSUE TITLE
Consider to drop supporting for RHEL 6 and SLES 11 management  and service node

### ISSUE THREAD

**Issue by neo954** [xcat2/xcat-core#5962]
This is a proposal of drop xCAT support for RHEL 6 and SLES 11. I am not talking about drop the support right now. Maybe the proper time point is when xCAT releases 2.14.6 or 2.15. At which point, xCAT will official add RHEL 8 support.

There are many good reasons to do this. I will list some of them below.

* RHEL 6 was first released on Nov 10, 2010. And SLES 11 was first release on Mar 24, 2009. They are outdated, and no more mainstream choice for new Linux operating system installation.
* We already made [some] [dirty] [patches] for SLES 11.
* When support for RHEL 8 was added to xCAT, many patches were added in a way that will break RHEL 6 and SLES 11 compatible. Which includes,
  * Use TLSv1.2 as the default setting for OpenSSL.
    [Industry] is actively following guidance provided by NIST and the PCI Council deprecating TLSv1.0 and TLSv1.1 by June 30, 2018. TLSv1.2 should remain a minimum baseline for TLS support at this time.
  * Drop support of SysV init script.
  * Drop support of `xinetd`.
  * Add support of `goconserver` and drop support of traditional `conserver`.

[some]: https://github.com/xcat2/xcat-core/pull/5522
[dirty]: https://github.com/xcat2/xcat-core/pull/5772
[patches]: https://github.com/xcat2/xcat-core/pull/5832
[Industry]: https://tools.ietf.org/id/draft-moriarty-tls-oldversions-diediedie-00.html

So, let us think about this proposal and try to make a decision before xCAT release 2.15.

**Comment by robin2008** [comment]
Note:  here is not remove RHEL 6 and SLES 11 support, but only drop RHEL 6/SLES 11 MN supporting.
The cross provision might still work, for example, RH7 -> RH6

**Comment by hu-weihua** [comment]
After comparing ``xcat-dep-201811190436.tar.bz2`` which are the official xcat-dep for xcat 2.14.5/2.14.6 with ``xcat-dep-rhel8.tar.bz2`` which plan to be released as 2.15 xcat-dep, there are below difference:

* Delete below directories.  That means xCAT cat not support rh5, rh6, sles10. sles11 as MN any more after xcat 2.15.
```
< |-- SL5
< |-- SL6
< |-- fedora12
< |-- fedora13
< |-- fedora14
< |-- fedora15
< |-- fedora8
< |-- fedora9
< |-- rh5
< |-- rh6
< |-- sles10
< |-- sles11
```

* deleted below useless files
```
|-- xCAT-UI-deps-2.7-1.noarch.rpm			      <
|-- xCAT-UI-deps-2.8-2.noarch.rpm			      <
|-- xCAT-genesis-x86_64-2.7.7-snap201301100842.noarch.rpm     <
```

* Rebuilt ``grub2-xcat``, will effect ``sles12+ppcle``, `` rh7+ppc64``, ``rh7+ppcle`` MN,  effect to install ``sles12+x86``, ``rh7+x86`` CN in mixed environment.
```
< |-- grub2-xcat-2.02-0.16.el7.snap201506090204.noarch.rpm
---
> |-- grub2-xcat-2.02-0.76.el7.snap201811290530.noarch.rpm

```

* Added directory rh8 to support RHEL8(x86 | ppc64le)
   Below packages rebuilt:
   * conserver
   * fping
   * goconserver
   * ipmitool
   * net-snmp
